README.md

```diff
@@ -588,7 +588,7 @@ Recaptcha.configure do |config|
   config.secret_key = '6Lc6BAAAAAAAAKN3DRm6VA_xxxxxxxxxxxxxxxxx'
   config.verify_url = 'https://hcaptcha.com/siteverify'
   config.api_server_url = 'https://hcaptcha.com/1/api.js'
-  config.response_limit = 100000
```

Comment:
that would break anyone already using that field, how about adding `response_minimum = 100` instead ?